internal/frr/templates/bfdprofile.tmpl

```diff
@@ -13,12 +13,13 @@
     echo-mode
     {{end -}}
     {{ if .profile.EchoInterval -}}
-    echo-interval {{.profile.EchoInterval}}
+    echo transmit-interval {{.profile.EchoInterval}}
+    echo receive-interval {{.profile.EchoInterval}}
```

Comment:
This should ideally be different commit?

Reply:
nope, it's actually the core of the commit "make the generated files compatible to 8.0+".
From 8.0 frr format has separate echo-intervals, to the point that frr 9+ is throwing an error when deleting echo-interval (the ci failure I got in metallb).

Let me modify the description of the commit to make it clearer.